### PR TITLE
Support capitalized Content-Type header

### LINKED ISSUE #31: Header key is case sensitive when inspecting content-type
Hi there, me again! One more small issue that just came up in my testing.

When inspecting the incoming request's `content-type` header to determine how to parse the request body, `headers.get('content-type')` returns `undefined` if the headers are capitalized (e.g., `Content-Type`). In my configuration, the capitalized form seems to be the default for API Gateway. This ultimately causes an empty string to be sent as the body, resulting in this error from Apollo:

```
POST body missing, invalid Content-Type, or JSON object has no keys.
```

It seems like checking for both lowercase and capitalized versions of the header should be the simple fix. I'd be happy to open a small PR for this one as well.

Thanks!

### PR DESCRIPTION
**Pull request by christiangaetano**
Incoming request headers are inspected to determine how to parse the request body. Currently, this inspection uses `headers.get('content-type')`, which returns `undefined` if the header key is capitalized (i.e., `Content-Type`). This falls back to checking for the capitalized header if the lowercase one is not found.

Fixes #31.

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -111,7 +111,10 @@ function isV2Event(event: GatewayEvent): event is APIGatewayProxyEventV2 {
 
 function normalizeV1Event(event: APIGatewayProxyEvent): HTTPGraphQLRequest {
   const headers = normalizeHeaders(event.headers);
```
Comment (trevor-scheer) [inline src/index.ts:113]: I think I'd be more interested in seeing this change happen within `normalizeHeaders`. Header keys are case-insensitive, so I think we should just downcase all the incoming header keys using `.toLocaleLowerCase()` as a part of the normalization process. Does that sound reasonable to you?

**Review by trevor-scheer — CHANGES_REQUESTED** [review]
Can you please add a patch changeset for this via the `npx changeset` command and commit that file as a part of your PR?

**Comment by trevor-scheer** [comment]
Once we get this in shape and landed I'll cut a patch release so we can get both of your PRs published. The effort is much appreciated and thanks for guinea pigging for us 😄

File: src/index.ts (hunk 1 above)
Comment (christiangaetano) [reply src/index.ts]: That makes total sense to me and is a much cleaner solution. Thank you for pointing me in the right direction! I just pushed up that change along with a changeset.

**Review by trevor-scheer — APPROVED** [review]
🎉

**Comment by trevor-scheer** [comment]
Thanks @christiangaetano! Just published `1.0.1` with your fixes.

**Pull request merged**